qtaf-core/src/test/java/de/qytera/qtaf/selenium/DriverTest.java

```diff
@@ -45,14 +44,15 @@ public void testFirefoxDriverInstantiation() {
         DriverFactory.clearDriver();
     }
 
-    //@Test(groups = {"driver"})
+    /** This test was commented out because selenium currently does not support the Opera browser **/
```

Comment:
Isn't it better to remove the whole opera driver implementation, cause we can't support it at the moment?

I'm not sure, but we have to check the documentation, if we write something about the opera support.

Reply:
I didn't want to remove the code yet because maybe in the future Opera will be supported again.

Reply:
I removed the Opera driver from the framework, because it seems Opera won't be supported by Selenium in near future